### ISSUE TITLE
[Bug] Frontend: extra top and left margin

### ISSUE THREAD

**Issue by infinity1729** [infinity1729/EventListing#4]
**Description**
In the frontend you can see extra margin towards left and top side of the page. Expected behaviour is to set the margin to be 0.

**Screenshot**
![image](https://user-images.githubusercontent.com/52048907/205499994-0140b92e-1c88-4efd-b178-f2179e4c1667.png)

**Comment by Adittya-Gupta** [comment]
Hey @infinity1729 I wanna work on this issue!! Can you assign it to me

**Comment by pranavajith** [comment]
Hey.. I'd like to take up this issue.
@infinity1729  could you please assign it to me, and also add the difficulty label?

**Comment by infinity1729** [comment]
Hey! for now I'll be assigning this issue to @pranavajith , but we'll be adding more issues tonight so maybe @Adittya-Gupta you can have a look at them. 
Happy Contributing! ✌

**Comment by infinity1729** [comment]
@pranavajith any updates?

**Comment by pranavajith** [comment]
Yup.. will be done before tomorrow afternoon.. my laptop is having a small issue.. will get it fixed by today evening..
Also working on my html and css skills.. not really an expert..

**Comment by sankooru14** [comment]
@infinity1729 Can you assign this issue to me?

**Comment by ankushj0801** [comment]
Can I work on this issue? @infinity1729

**Comment by infinity1729** [comment]
@pranavajith any further updates? If you are stuck somewhere you may ask for help

**Comment by Tushar98644** [comment]
if no one is working on this issue ,may i take this up?

**Comment by Tushar98644** [comment]
@infinity1729   if no one is working on this issue ,may i take this up?

**Comment by infinity1729** [comment]
Alright assigning to @Tushar98644

**Comment by infinity1729** [comment]
@Tushar98644 any updates?

**Comment by Tushar98644** [comment]
a bit preoccupied at the moment...will complete by eod..

**Comment by anirudhhhh** [comment]
@infinity1729 may i take the issue if not done?
